crates/sui-core/src/epoch/reconfiguration.rs

```diff
@@ -197,20 +197,18 @@ where
                 std::str::from_utf8(net_addr).map_err(|e| SuiError::GenericAuthorityError {
                     error: e.to_string(),
                 });
-            let address: Multiaddr = str_addr
-                .unwrap()
-                .parse()
-                .map_err(|e: multiaddr::Error| SuiError::GenericAuthorityError {
+            let address: Multiaddr = str_addr.unwrap().parse().map_err(|e: multiaddr::Error| {
+                SuiError::GenericAuthorityError {
                     error: e.to_string(),
-                })
-                .unwrap();
-
-            let channel = net_config
-                .connect_lazy(&address)
-                .map_err(|e| SuiError::GenericAuthorityError {
-                    error: e.to_string(),
-                })
-                .unwrap();
+                }
+            })?;
```

Comment:
This may not be sufficient to fix the issue. We will never be able to go pass this error, so it's not fundamentally different from a panic.
There is no rush, worth taking the time and coming up with a proper solution.
I think we either need to make sure all addresses on-chain are valid upon creation, or we can tolerate errors here by skipping certain validators from the committee (prefer the former).